converters/fromZigbee.js

```diff
@@ -8575,6 +8575,71 @@ const converters = {
             return result;
         },
     },
+    zb_sm_tuya_cover: {
```

Comment:
Could you move both the added toZigbee and fromZigbee converter into `tuya.js` (`fzLocal`/`tzLocal`) We have problems of these files getting too big.

Reply:
ok